neuraxle/pipeline.py

```diff
@@ -131,18 +131,18 @@ def load(self, pipeline: 'Pipeline', data_container: DataContainer) -> 'Pipeline
         pipeline_cache_folder = os.path.join(self.cache_folder, pipeline.name)
         pipeline_cache_list_file_name_path = os.path.join(pipeline_cache_folder, self.pipeline_cache_list_file_name)
         if not os.path.exists(pipeline_cache_list_file_name_path):
-            return pipeline
+            return None
 
         with open(pipeline_cache_list_file_name_path, mode='r') as file:
             found_cached_pipeline = False
             for line in file.readlines():
                 if found_cached_pipeline:
-                    return load(line)
+                    return load(line.strip())
```

Comment:
really? this `strip` surprises/startles me, but I don't know why.

Reply:
it is needed to remove \n